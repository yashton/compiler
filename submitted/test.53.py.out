(LIT "")
(LIT "long string")
(KEYWORD with)
(ID "quote")
(ID "char")
(LIT "")
(ERROR "Illegal character '"
' on line 1)
