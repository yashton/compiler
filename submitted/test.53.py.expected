(LIT "long string\" with quote char")
(NEWLINE)
(ENDMARKER)
